config/prod/prow/jobs/custom-jobs.yaml

```
@@ -39,33 +39,33 @@ presubmits:
           secretName: repoview-token
   - name: pull-test-infra-validate-prow-yaml
     decorate: true
-    run_if_changed: "^config/prow/(cluster|core|jobs)/.*.yaml$"
+    run_if_changed: "^config/prod/prow/(cluster|core|jobs)/.*.yaml$"
     cluster: "build-knative"
     spec:
       containers:
       - image: gcr.io/k8s-prow/checkconfig:v20200413-65958d4a8
         command:
         - "/checkconfig"
         args:
-        - "--config-path=config/prow/core/config.yaml"
-        - "--job-config-path=config/prow/jobs"
-        - "--plugin-config=config/prow/core/plugins.yaml"
+        - "--config-path=config/prod/prow/core/config.yaml"
+        - "--job-config-path=config/prod/prow/jobs"
+        - "--plugin-config=config/prod/prow/core/plugins.yaml"
         - "--prow-yaml-repo-name=$(REPO_OWNER)/$(REPO_NAME)"
         - "--strict"
         - "--exclude-warning=mismatched-tide"
   - name: pull-test-infra-validate-staging-prow-yaml
     decorate: true
-    run_if_changed: "^config/prow-staging/(cluster|core|jobs)/.*.yaml$"
+    run_if_changed: "^config/(prod/staging)/prow/(cluster|core|jobs)/.*.yaml$"
```

This job should only be run if staging Prow config is changed? So
```suggestion
    run_if_changed: "^config/staging/prow/(cluster|core|jobs)/.*.yaml$"
```
?